<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pizzamenu</class>
 <widget class="QWidget" name="pizzamenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#pizzamenu{
border-image: url(:/newPrefix/piza.png);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 18pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 18pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>ADD MORE</string>
   </property>
   <property name="icon">
    <iconset resource="resim/Yeni Metin Belgesi.qrc">
     <normaloff>:/newPrefix/more.ico</normaloff>:/newPrefix/more.ico</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 18pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>CART</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resim/cart.ico</normaloff>resim/cart.ico</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 26, 19);
color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
color: rgb(246, 246, 246);
border-radius: 10px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 26, 19);
color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
color: rgb(246, 246, 246);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 26, 19);
color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
color: rgb(246, 246, 246);
border-radius: 10px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
color: rgb(246, 246, 246);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(246, 246, 246);</string>
   </property>
   <property name="text">
    <string>VEG PIZZA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(246, 246, 246);</string>
   </property>
   <property name="text">
    <string>MIX PIZZA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(246, 246, 246);</string>
   </property>
   <property name="text">
    <string>MARGARITA PIZZA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(246, 246, 246);</string>
   </property>
   <property name="text">
    <string>CHICKEN PIZZA</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>520</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 246, 246);
background-color: rgb(0, 85, 127);
border-radius: 10px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton{
color: rgb(246, 246, 246);
background-color: rgb(0, 85, 127);
border-radius: 10px;

}
.QPushButton:hover
{
 	background-color: rgb(0, 150, 127);
}
</string>
   </property>
   <property name="text">
    <string>Total Cost</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resim/money.png</normaloff>resim/money.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 246, 246);
background-color: rgb(43, 26, 19);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(246, 246, 246);

border-radius: 10px;
font: 20pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>KIND</string>
   </property>
   <property name="icon">
    <iconset resource="resim/Yeni Metin Belgesi.qrc">
     <normaloff>:/newPrefix/p.ico</normaloff>:/newPrefix/p.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(246, 246, 246);
border-radius: 10px;

border-radius: 10px;
font: 20pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>COST</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resim/money.png</normaloff>resim/money.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(246, 246, 246);
border-radius: 10px;

border-radius: 10px;
font: 20pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>QUANTITY</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resim/Yeni Metin Belgesi.qrc"/>
 </resources>
 <connections/>
</ui>
